gui/control_main.py

```diff
@@ -2220,13 +2243,18 @@ def processRingCurrent(self, ringCurrentVal):
         else:
             self.ringCurrentMessage.setStyleSheet("background-color: #99FF66;")
 
-    def processBeamAvailable(self, beamAvailVal):
-        if int(beamAvailVal) == 1:
+    '''
```

Comment:
beam available is supposed to be a global signal of whether the sample would be exposed to X-rays if the shutter were open, regardless of what the user is doing on the GUI.